gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -335,13 +335,21 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         showProjectSelector();
         closeDrawer();
         break;
+      case R.id.nav_download_tiles:
+        showTileDownloadLayer();
```

Comment:
Naming nit: This isn't a layer from the user's perspective, it's a UI for selecting and deselecting offline satellite imagery. Also, the fact that users select tiles is for the most part an implementation detail; we could in the future allow people to choose an area instead of individual tiles. In that case, I'd probably call it something like "offline imagery selector" or similar.

Reply:
That makes sense, I'll rename it!